posthog/hogql/property.py

```diff
@@ -642,6 +652,15 @@ def property_to_expr(
     )
 
 
+def create_expr_for_revenue_analytics_property(property: RevenueAnalyticsPropertyFilter) -> ast.Expr:
+    if property.key == "product":
+        return ast.Field(chain=[RevenueAnalyticsProductView.get_generic_view_alias(), "name"])
+    elif property.key == "amount":
+        return ast.Field(chain=[RevenueAnalyticsInvoiceItemView.get_generic_view_alias(), "amount"])
+    else:
+        raise QueryError(f"Revenue analytics property filter key {property.key} not implemented")
```

Comment:
Kinda bad, I know, but it's the only possible way to solve this since these come from several different tables depending on the value. I attempted using a map or a JSON structure but it's much trickier/worse than doing this IMO.